easybuild/easyblocks/generic/conda.py

```diff
@@ -57,11 +59,19 @@ def extract_step(self):
             super(Conda, self).extract_step()
 
     def install_step(self):
-        """Install software using 'conda env create' or 'conda create' & 'conda install'."""
+        if (get_software_root('anaconda2') or get_software_root('miniconda2') or
```

Comment:
```suggestion
        """Install software using 'conda env create' or 'conda create' & 'conda install' (or the 'mamba', etc., equivalent)."""
        if (get_software_root('anaconda2') or get_software_root('miniconda2') or
```
Please keep a docstring around